internal/app/machined/pkg/runtime/configurator.go

```diff
@@ -2,48 +2,77 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-package machine
+package runtime
```

Comment:
this implies that this interface is protected interface of the app, while certainly many places consume say machine type?